{"hierarchy":{"paths":[["doc:\/\/YubiKit\/documentation\/YubiKit","doc:\/\/YubiKit\/documentation\/YubiKit\/ConnectionError"]]},"primaryContentSections":[{"declarations":[{"languages":["swift"],"platforms":["macOS"],"tokens":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"cancelledByUser","kind":"identifier"}]}],"kind":"declarations"}],"schemaVersion":{"minor":3,"patch":0,"major":0},"identifier":{"url":"doc:\/\/YubiKit\/documentation\/YubiKit\/ConnectionError\/cancelledByUser","interfaceLanguage":"swift"},"metadata":{"role":"symbol","symbolKind":"case","externalID":"s:7YubiKit15ConnectionErrorO15cancelledByUseryA2CmF","fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"cancelledByUser","kind":"identifier"}],"modules":[{"name":"YubiKit"}],"title":"ConnectionError.cancelledByUser","roleHeading":"Case"},"kind":"symbol","sections":[],"abstract":[{"type":"text","text":"Awaiting call to connect() was dismissed by the user."}],"variants":[{"paths":["\/documentation\/yubikit\/connectionerror\/cancelledbyuser"],"traits":[{"interfaceLanguage":"swift"}]}],"references":{"doc://YubiKit/documentation/YubiKit":{"images":[{"type":"icon","identifier":"documentation-header.png"}],"url":"\/documentation\/yubikit","identifier":"doc:\/\/YubiKit\/documentation\/YubiKit","kind":"symbol","type":"topic","role":"collection","abstract":[{"text":"Connect and run commands on the different applications on a YubiKey. The framework support connecting using NFC, Lightning and USB-C.","type":"text"}],"title":"YubiKit"},"doc://YubiKit/documentation/YubiKit/ConnectionError":{"type":"topic","role":"symbol","title":"ConnectionError","identifier":"doc:\/\/YubiKit\/documentation\/YubiKit\/ConnectionError","url":"\/documentation\/yubikit\/connectionerror","navigatorTitle":[{"text":"ConnectionError","kind":"identifier"}],"kind":"symbol","fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"text":"ConnectionError","kind":"identifier"}],"abstract":[{"text":"Connection Errors.","type":"text"}]},"doc://YubiKit/documentation/YubiKit/ConnectionError/cancelledByUser":{"type":"topic","role":"symbol","title":"ConnectionError.cancelledByUser","identifier":"doc:\/\/YubiKit\/documentation\/YubiKit\/ConnectionError\/cancelledByUser","url":"\/documentation\/yubikit\/connectionerror\/cancelledbyuser","kind":"symbol","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"cancelledByUser"}],"abstract":[{"type":"text","text":"Awaiting call to connect() was dismissed by the user."}]},"documentation-header.png":{"identifier":"documentation-header.png","type":"image","variants":[{"traits":["2x","light"],"url":"\/images\/YubiKit\/documentation-header@2x.png"},{"traits":["2x","dark"],"url":"\/images\/YubiKit\/documentation-header~dark@2x.png"}],"alt":"YubiKeys in a row"}}}